docs/tutorials/backport.md

```diff
@@ -1,21 +1,33 @@
 !!! warning "This tutorial might be partial or incomplete. Please check with the team if you have questions."
 
-Before the existance of the ETL, datasets were directly uploaded to our Grapher admin site. Sometimes, we want to use these datasets in ETL. However, they are not available by default as they were never imported via the ETL. To make them available in ETL we use _backport_.
+Before ETL came along, datasets were uploaded directly to our Grapher admin site. If you want to leverage these datasets in the ETL, they won't be available out of the box, since they weren't initially imported via ETL. You have two main options to bring them into ETL:
 
+1. Migrate them into the ETL pipeline.
+2. Move them to Fast-Track.
 
-The first step is getting them to Snapshot using:
+Choose Fast-Track if the dataset requires author edits. Otherwise, it's more straightforward to migrate to ETL.
```

Comment:
This sounds as if fast-track gives you more freedom than migrating to ETL, which is not true. It's not clear to me what the benefit of fast-track is. Maybe "Fast-Track is recommended for users who prefer working on Google sheets rather than the ETL environment." (e.g. authors). Is this right?

Reply:
You're right, I've clarified that.